research/gnn-survey/layers.py

```diff
@@ -183,3 +185,74 @@ def call(self, inputs):
 
     output = tf.sparse.sparse_dense_matmul(attn, x)
     return output
+
+
+class GraphIsomorphismLayer(tf.keras.layers.Layer):
+  """Single graph isomorphism layer."""
+
+  def __init__(self, mlp_layers, output_dim, dropout_rate, learn_eps=False, sparse=True, **kwargs):
+    """Initializes the GraphIsomorphismLayer.
+
+    Args:
+      mlp layers: (int) Number of mlp layers
+      output_dim: (int) Output dimension of gcn layer
+      dropout_rate: (float) Dropout probability
+      learn_eps: (bool) Whether to learn the epsilon weighting
+      sparse: (bool) sparse: Whether features are sparse
+      **kwargs: Keyword arguments for tf.keras.layers.Layer.
```

Comment:
Enclose in backticks.

Reply:
Do you mean that enclose `tf.keras.layers.Layer` in backticks?

Reply:
Yes, thanks.